Lapin-Learn/backend-main-server
# feat(lap-343): create pwd account before log in with provider

1. Make sure no account with test email in database and firebase
2. Test:
- log in with provider at first
- check firebase whether the pwd account is also created with the same email
- use forgot password to change password of that email
- log in with email and new password again
